onnxruntime/wasm/api.cc

```diff
@@ -0,0 +1,89 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "api.h"
+
+#include "core/session/onnxruntime_cxx_api.h"
+
+#include <vector>
+
+namespace {
+Ort::Env* g_env;
+}  // namespace
+
+void ort_init() {
+  OrtLoggingLevel logging_level = ORT_LOGGING_LEVEL_WARNING;
+  g_env = new Ort::Env{logging_level, "Default"};
```

Comment:
Is there a way to change the logging level if a user needs more diagnostic output to investigate an issue?

Reply:
I put a comment in the code as a TODO item.